templates/ecr/README.md

```diff
@@ -17,6 +17,10 @@ Run this container with the following environment variables:
 * `AWS_ACCESS_KEY_ID`
 * `AWS_SECRET_ACCESS_KEY`
 
+If you're using an IAM instance profile, then you only need to specify the region,
+with the `AWS_REGION` environment variable, and the credentials will be
+automatically negotiated.
+
```

Comment:
Can please update this file by just copying the contents from https://raw.githubusercontent.com/objectpartners/rancher-ecr-credentials/master/README.md